raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -155,6 +155,10 @@ describe('Raiden', () => {
   const txBlock = 42;
   const txHash = makeHash();
   const transferAmount = 42;
+  const id = 10;
+  const settleTimeout = 500;
+  const isFirstParticipant = true;
+  const channelOpenHash = makeHash();
```

Comment:
Can you use `txHash` here instead of creating a new one?

Reply:
I have named all of them differently as well declared them differently so that there is no ambiguity to anyone reading the tests later.